scripts/services/north_C

```diff
@@ -10,5 +10,28 @@ if [ ! -d "${FLEDGE_ROOT}" ]; then
 	exit 1
 fi
 
+runvalgrind=n
+if [ "$VALGRIND_NORTH" != "" ]; then
+	for i in "$@"; do
+		case $i in
+			--name=*)
+				name=i"`echo $i | sed -e s/--name=//`"
```

Comment:
Fixed in https://github.com/fledge-iot/fledge/pull/1171/commits/2685db1ae0dc6628218c0b9385d88b5682e67213